app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The association should be `belongs_to :author, class_name: 'User'`

Reply:
* This hasn't been treated as stated by the previous reviewer kindly fix this 🙏